configure.ac

```diff
@@ -81,13 +81,26 @@ SP_CHECK_CXXFLAGS_APPEND([OUR_CXXFLAGS], [\
 AC_ARG_ENABLE(
     [vma-api],
     AC_HELP_STRING([--enable-vma-api],
-                   [SOCKPERF: enable vma extra api support (default=no)]),
+                   [SOCKPERF: enable vma extra api support: 'yes', 'no' or library installation path (default=no)]),
     [have_vma_api=$enableval],
     [have_vma_api=no])
-AS_IF([test "x${have_vma_api}" == "xyes"],
-    [AC_CHECK_HEADER([mellanox/vma_extra.h],
+AS_IF([test "${have_vma_api}" != "no"],
+    [
+    if test "$have_vma_api" = "yes"
+    then
+        have_vma_api=/usr
+    fi
+    CPPFLAGS="$CPPFLAGS -I$have_vma_api/include"
+    if test -d "$have_vma_api/lib64"
```

Comment:
no reason to check and assign to LDFLAGS library path for compilation. Pointing to header file is enough.

Reply:
added a comment to RM #3112201 to fix it later